### PR TITLE
Initial clean up of project statistics popup

### LINKED ISSUE #1011: Clean up project statistics popup
After we introduce new graphs, tables and styling on project statistic popup it would be nice to finalize and clean up:
* check that styling is consistent
* check if there are any bugs or any things to improve
* refactor and clean up the code e.g. remove (or merge) duplicating styles...

### PR DESCRIPTION
**Pull request by vasily-pozdnyakov**
### Summary of changes

* Changes in styling
* Pie chart extracted into a separate component to use it for next pie charts
* Refactoring

### Context and reason for change

Need to prepare for introduction of more pie charts.

Initial fix #1011

Before:
image

After:
image

### CODE REVIEW

**Comment by JakobSchubert** [comment]
For changes in styling, before/after screenshots would be helpful for the reviewer.

**Review by FlorianSchepers — CHANGES_REQUESTED** [review]

File: src/Frontend/Components/ProjectStatisticsPopup/PieCharts.tsx [hunk 1]
```diff
@@ -5,84 +5,59 @@
...
-            nameKey="licenseName"
+            nameKey="name"
             cx="50%"
             cy="50%"
             paddingAngle={1}
             minAngle={5}
-            outerRadius="65%"
-            isAnimationActive={false}
-            label={renderCustomizedLabel}
+            outerRadius="70"
```
Comment (JakobSchubert) [inline src/Frontend/Components/ProjectStatisticsPopup/PieCharts.tsx:52]:
```suggestion
            outerRadius={70}
```
should also work and is more consistent with the other number values like `minAngle`.

**Review by JakobSchubert — APPROVED** [review]
Looks good. Some small remarks.

File: src/Frontend/enums/enums.ts [hunk 2]
```diff
@@ -80,3 +80,11 @@ export enum CheckboxLabel {
   FollowUp = 'Follow-up',
   ExcludeFromNotice = 'Exclude From Notice',
 }
+
+export enum ProjectStatisticsPopupTitles {
+  AttributionCountPerSourcePerLicenseTable = 'Signals per Sources',
+  AttributionPropertyCountTable = 'First Party and Follow Up Attributions',
+  CriticalLicensesTable = 'Critical Licenses',
+  MostFrequentLicenseCountPieChart = 'Most Frequent Licenses',
+  AttributionCountByCriticalityPieChart = 'Attributions per Criticality',
```
[the remark this answers is not in the document]
Comment (JakobSchubert) [reply src/Frontend/enums/enums.ts:89]: To be more precise: `AttributionCountByCriticalityPieChart` is unused

**Comment by JakobSchubert** [comment]
Optional remark: The legend for the pie chart is a bit far away from the actual pie chart. One could think about either moving it closer to the pie chart or adding a box around the whole (pie chart + legend) to make it clear to the viewer that the legend belongs to the pie chart.

File: src/Frontend/Components/ProjectStatisticsPopup/CriticalLicensesTable.tsx [hunk 3]
```diff
@@ -135,7 +135,7 @@ export function CriticalLicensesTable(
                   key={index}
                   align={index === 0 ? 'left' : 'center'}
                 >
-                  {columnName.toUpperCase()}
+                  {columnName}
```
[the remark this answers is not in the document]
Comment (vasily-pozdnyakov) [reply src/Frontend/Components/ProjectStatisticsPopup/CriticalLicensesTable.tsx:138]: Actually here it is not really necessary, as there are only two defined columns, which are developer-defined.

File: src/Frontend/Components/ProjectStatisticsPopup/project-statistics-popup-helpers.ts [hunk 4]
```diff
@@ -34,8 +34,8 @@ const ATTRIBUTION_PROPERTIES_TO_DISPLAY: Array<keyof PackageInfo> = [
 export const SOURCE_TOTAL_Key = 'Total';
 export const LICENSE_TOTAL_Key = 'Total';
 export const ATTRIBUTION_TOTAL_Key = 'Total Attributions';
-export const LICENSE_COLUMN_NAME_IN_TABLE_Key = 'License';
-export const TOTAL_COLUMN_NAME_IN_TABLE_Key = 'Total';
```
[the remark this answers is not in the document]
Comment (vasily-pozdnyakov) [reply src/Frontend/Components/ProjectStatisticsPopup/project-statistics-popup-helpers.ts:38]: Honestly, I also do not get it. Will change.

**Review by FlorianSchepers — APPROVED** [review]

**Pull request merged**